lucupy/sky/utils.py

```diff
@@ -124,7 +124,7 @@ def min_max_alt(lat: Angle, dec: AngleParam) -> Tuple[Angle, Angle]:
     return Angle(minalt, unit=u.rad), Angle(maxalt, unit=u.rad)
 
 
-def local_midnight_time(time: Time, localtzone: BaseTzInfo) -> Time:
+def local_midnight_time(time: Time, localtzone: timezone) -> Time:
```

Comment:
This should trigger mypy. timezone is not a type is a function, correct type is BaseTzInfo

Reply:
Yes... this is weird indeed... I didn't even ask it to do this, and I did a pull before.